fe/fe-core/src/main/java/org/apache/doris/job/manager/JobManager.java

```diff
@@ -328,6 +373,18 @@ public void replayUpdateJob(T job) {
                 .add("msg", "replay update scheduler job").build());
     }
 
+    public void replayUpdateStreamingJob(AlterStreamingJobOperationLog log) {
```

Comment:
Can the update operations for streaming jobs and other jobs be unified without adding a new AlterStreamingJobOperationLog?

Reply:
Because other jobs are currently played back globally, and the offset and other information of streaming are stored in txn, the status of playback modification will overwrite the offset and other information.